apps/sms/views/conversation/js/attachment_renderer.js

```diff
@@ -203,6 +203,16 @@
 
               // It's essential to remember real image URL to revoke it later
               attachmentContainer.dataset.thumbnail = data.thumbnail.url;
+
+              var fileNode = contentContainer.querySelector('.file-name');
+              if (this._attachment.name) {
+                fileNode.removeAttribute('data-l10n-id');
+                fileNode.textContent = this._attachment.name.slice(
+                  this._attachment.name.lastIndexOf('/') + 1
```

Comment:
I don't know if I understand. Do you want `getBaseMarkup` to figure out if there is this._attachment.name and either get it's value from `formatValue` or do the sliceing on the name?